### PR TITLE
Updated Default project template

### LINKED ISSUE #5732: Default template has a double heading/title
### Steps to Reproduce

1. File | New | Maui
2. Run
3. Observe a double title bar

### Version with bug

Release Candidate 1

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, Android, Windows, macOS

### Affected platform versions

All

### Did you find any workaround?

One workaround (maybe the correct correct thing?) is to use a `Shell.NavigationBarIsVisible="False"` to hid the OS one.

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change

This will avoid confusion with the double title. @davidortinau What should be the new title?, ".NET MAUI" is fine?

image

image

### Issues Fixed

Fixes #5732

### CODE REVIEW

**Review by rmarinho — APPROVED** [review]

File: src/Templates/src/templates/maui-mobile/AppShell.xaml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     Shell.FlyoutBehavior="Disabled">
 
     <ShellContent
-        Title="Hello, World!"
+        Title=".NET MAUI"
```
Comment (rachelkang) [inline src/Templates/src/templates/maui-mobile/AppShell.xaml:10]: Maybe ".NET MAUI App"

Or ideally, is there a way we could directly reference the App Title here?

**Review by rachelkang — CHANGES_REQUESTED** [review]

**Comment by rachelkang** [comment]
@davidortinau This looks great, thanks!

One question - what about the current count label? Should we keep that above the "Click me" button? (It doesn't make sense to remove it completely while keeping the SemanticReader.Announce logic, unless we reove both and make the button do somethng else)

**Comment by davidortinau** [comment]
> @davidortinau This looks great, thanks!
> 
> One question - what about the current count label? Should we keep that above the "Click me" button? (It doesn't make sense to remove it completely while keeping the SemanticReader.Announce logic, unless we reove both and make the button do somethng else)

The button text updates now instead of the label. 

image

**Comment by rachelkang** [comment]
@davidortinau Sorry, I see it now that you added it to update the button text

**Comment by Redth** [comment]
@jsuarezruiz  @rachelkang not sure which colors we wanted in the conflict here, can you resolve them please?

**Pull request merged**